docs/BEMAN_STANDARD.md

```diff
@@ -105,6 +105,51 @@ with a relevant set of codeowners, starting with the local codeowners and ending
 Known exceptions:
 * [mpark/wg21: Framework for Writing C++ Committee Papers](https://github.com/mpark/wg21): A non-C++ submodule designed for drafting ISO C++ papers using LaTeX or Markdown.
 
+## Release
+
+**[RELEASE.GITHUB]**: REQUIREMENT: All Beman libraries must be released using [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github).
+
+
+**[RELEASE.NOTES]**: RECOMMENDATION: Release should include:
+
+1. A title that clearly indicates the release version and purpose.
+2. An "About" section describing the main purpose and impact of the release.
+3. A "Release Maturity" section indicating the current status of the library according to the [Beman Library Maturity Model](https://github.com/bemanproject/beman/blob/main/docs/BEMAN_LIBRARY_MATURITY_MODEL.md).
+4. A "What's Changed" section listing all changes, organized by:
+   - Breaking changes (if any)
+   - New features and improvements
+   - Bug fixes
+   - Build system changes
+   - Documentation updates
+5. A "New Contributors" section listing all first-time contributors.
+6. A "Full Changelog" link to the complete list of commits.
+7. A "Contributors" section listing all contributors to this release. Note: This section is autogenerated by GitHub Releases!
+
+Use the following format:
+
+```markdown
```

Comment:
Should we also had a placeholder for the title?

Reply:
No. Updated the docs (`Check visual example for the field location.`) and the visual example.